images/go/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM golang:1.9-alpine
+LABEL maintainer="linterhub@repometric.com"
+RUN apk update && apk upgrade && apk add git
```

Comment:
I suggest to clean up the cache [documentation](from https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run) and install only required dependencies (e.g. we don not need `git` on this layer).

Reply:
I installed `git` on this layer because it's required for `go get` to work, and I figured we'd want `go get` to work everywhere. Where would you suggest I install it?